javascript/src/main/components/ChannelMessages/MessageScrollableView.js

```diff
@@ -24,11 +24,17 @@ const formatBracketedText = (text) => {
         for (let i=0; i<found.length; i++){
             var current = found[i].replace('<', '');
             current = current.replace('>', '');
-            if (found[i].includes("|") && (found[i].includes("mailto") || found[i].includes("http"))){
+            if (found[i].includes("|") && (found[i].includes("mailto") || found[i].includes("http") || found[i].includes("tel"))){      // embedded links
                 var links = current.split('|');
                 text = text.replace(found[i], '<a href = ' + links[0] + ' target = "_blank">' + links[1] +'</a>')
-            }else if (found[i].includes("http") || found[i].includes("mailto")){
+            }else if (found[i].includes("http") || found[i].includes("mailto") || found[i].includes("tel")){                            // unembedded links
                 text = text.replace(found[i], '<a href = ' + current + ' target = "_blank">' + current + '</a>')
+            }else if (found[i].includes("|")){                                                                                          // channel links
+                links = current.split('|');
+                text = text.replace(found[i], '<a href = /member/channels/' + links[1] + '>' + links[1] + '</a>')
```

Comment:
Quick change: Can you add the # in front of the text for the channel link so it looks different than a regular link format

Reply:
Sure thing!

Reply:
I added the '#', you can test it out here: https://cs156-w21-team-6pm-2-mapache.herokuapp.com/ .